RevenueCatUI/Helpers/Package+VariableDataProvider.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import RevenueCat
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+extension Package: VariableDataProvider {
+
+    var localizedPricePerMonth: String {
+        return self.priceFormatter.string(from: self.pricePerMonth) ?? ""
+    }
+
+    var productName: String {
+        return self.storeProduct.localizedTitle
+    }
+
+}
+
+// MARK: - Private
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+private extension Package {
+
+    var pricePerMonth: NSDecimalNumber {
+        guard let price = self.storeProduct.pricePerMonth else {
+            fatalError("Unexpectedly found a package which is not a subscription: \(self)")
```

Comment:
Is `fatalError` a thing we do? Does this crash the app? I feel like this is a thing I should know lol

Reply:
I forgot to bring this up. Yeah we need to discuss how each template configures the displayed packages. Before we get to this point we need to make sure this is a subscription.

Reply:
This will be solved by what we just discussed. By specifying package (types), we ensure that only subscriptions come here.

Reply:
Done in #2798.